lib/brcobranca/remessa/cnab400/bradesco.rb

```diff
@@ -83,17 +83,17 @@ def monta_detalhe(pagamento, sequencial)
           detalhe << ''.rjust(1, '0')                                 # digito da conta corrente (op)               X[01]       020 a 020
           detalhe << identificacao_empresa                            # identficacao da empresa                     X[17]       021 a 037
           detalhe << ''.rjust(25, ' ')                                # num. controle                               X[25]       038 a 062
-          detalhe << cod_banco                                        # codigo do banco                             9[03]       063 a 065
+          detalhe << ''.rjust(3, '0')                                 # codigo do banco (debito automatico apenas)  9[03]       063 a 065
```

Comment:
Line is too long. [137/80]<br>Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.